kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1646,10 +1646,12 @@ object KyuubiConf {
     .createWithDefault(-1)
 
   val ENGINE_POOL_BALANCE_POLICY: ConfigEntry[String] =
-    buildConf("kyuubi.engine.pool.balance.policy")
-      .doc("The balance policy of queries in engine pool.<ul>" +
-        " <li>RANDOM - Randomly use the engine in the pool</li>" +
-        " <li>POLLING - Polling use the engine in the pool</li> </ul>")
+    buildConf("kyuubi.engine.pool.selectPolicy")
+      .doc("The policy of selecting engine in a pool engine for session." +
```

Comment:
The select policy of an engine from the corresponding engine pool engine for a session

Reply:
updated